Drop spec table from “Feature-Policy: sync-xhr” article

sideshowbarker opened the pull request:
https://github.com/whatwg/xhr/pull/322 removed the `sync-xhr` policy-controlled feature from XHR spec — so this change removes the Specifications table from the “Feature-Policy: sync-xhr” article.

sideshowbarker committed:
Drop spec table from “Feature-Policy: sync-xhr”

https://github.com/whatwg/xhr/pull/322 removed the “sync-xhr”
policy-controlled feature from XHR spec — so this change removes the
Specifications table from the “Feature-Policy: sync-xhr” article.

domenic commented:
I'm not sure deprecated is right here: it's rather experimental, like many other only-one-implementer features.

sideshowbarker commented:
> I'm not sure deprecated is right here: it's rather experimental, like many other only-one-implementer features.

You’re right. So I’ve reverted the part of the change that added the Deprecated header and that instead has SeeCompatTable — to point readers at the Browser Compatibility table — and instead kept just the part of the change that drops the Specifications table. (And updated the PR description here, and the commit message.)

sideshowbarker commented:
@mirunacurtean I think this one is ready for re-review and merging
